src/main/java/ch/njol/skript/sections/EffSecSpawn.java

```diff
@@ -58,9 +59,10 @@
 	"spawn 3 creepers at the targeted block",
 	"spawn a ghast 5 meters above the player",
 	"spawn a zombie at the player:",
-	"\tset name of the zombie to \"\""
+		"\tset name of the zombie to \"\""
 })
-@Since("1.0, 2.6.1 (with section)")
+@Since("1.0, 2.6.1 (with section), INSERT_VERSION (dropped items)")
+@RequiredPlugins("Minecraft 1.17.1+ (dropped items)")
```

Comment:
Is this still needed? Since you do it manually in case there isn't a viable method available

Reply:
Oh yes thanks